### PR TITLE
[14.0][FIX] update copier template

### PR DESCRIPTION
**Pull request by Saran440**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+
```
Comment (MiquelRForgeFlow) [inline .pylintrc]: avoid white spaces

File: .pylintrc-mandatory [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (MiquelRForgeFlow) [inline .pylintrc-mandatory]:
```suggestion
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by Saran440** [comment]
@MiquelRForgeFlow Thanks for review, i fix it BUT other repo is space too. may be it unnecessary.

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**